src/Servers/Kestrel/Transport.Sockets/src/SocketsStrings.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="FileHandleEndPointNotSupported" xml:space="preserve">
+    <value>The Socket transport does not support FileHandleEndPoints. Consider using the libuv transport instead.</value>
```

Comment:
I'm happy to change the verbiage if anyone has suggestions for improvement.

Reply:
Does not support binding to file handles.

Reply:
```suggestion
    <value>The Socket transport does not support binding to file handles. Consider using the libuv transport instead.</value>
```